src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx

```diff
@@ -61,7 +70,25 @@ export default class OrganizationProjectsView extends AsyncView {
     return `${org.name} Projects`;
   }
 
-  onSearch = e => {
+  handleChange = evt => {
+    let searchQuery = evt.target.value;
+    this.setState({searchQuery}, this.getProjects);
```

Comment:
Looking at this again (and this applies to members too), I think I led you down the wrong path. While this works, there can be a longer than necessary delay before search occurs since we are both: waiting for state to update, and a debounce.

The better way is to remove `getProjects`'s dependency on state (since there's really no reason tie it to `state`) and just pass in the search query as an argument.